styles/styles.css

```diff
@@ -15,6 +15,11 @@
   src: local("Arial");
 }
 
+@font-face {
```

Comment:
do we really need this font? which icon do you need? you could do inline SVG instead.

Reply:
It is for symbols used in mobile view navigation. I used the same font used currently for that in mammotome.com/

![Screenshot_2023-03-27_at_16_50_55](https://user-images.githubusercontent.com/1773950/227977321-730d3373-7ddd-4ad9-8ad2-47a9c73bedd9.png)

Reply:
i understand. so far we've been using inline SVG to save a request and inclusion of a font file, of which we may only use a fraction. better ideas welcome.